tests/beaker_tests/cisco_vlan/vlan/standardvlan_provider_nondefaults.rb

```diff
@@ -99,10 +99,11 @@
       "resource cisco_vlan '128'", options)
     on(agent, cmd_str) do
       search_pattern_in_output(stdout,
-                               { 'ensure'    => 'present',
-                                 'shutdown'  => 'true',
-                                 'state'     => 'suspend',
-                                 'vlan_name' => 'DESCR-VLAN0128' },
+                               { 'ensure'     => 'present',
+                                 'mapped_vni' => '128000',
```

Comment:
Couple thoughts here.

1) We need to update all of the test files under `tests/beaker_tests/cisco_vlan/vlan/` to check for default and non-default cases.
2) These tests run on multiple nexus test beds so we will need to build the manifest and test hashes based on the platform.  Don't create separate hashes but rather modify the existing hashes based on platform.
3) You can use the `platform` api to detect the platform and do the right thing.
4) You can modify any methods that you are using in https://github.com/cisco/cisco-network-puppet-module/blob/release_1.2.0/tests/beaker_tests/cisco_vlan/vlan/vlanlib.rb to pass the platform info.